src/helper/storage/handler.js

```diff
@@ -3,9 +3,12 @@ import DummyStorage from './dummy';
 import CookieStorage from './cookie';
 import Warn from '../warn';
 
-function StorageHandler() {
+function StorageHandler(options) {
   this.warn = new Warn({});
   this.storage = new CookieStorage();
+  if (!options.__tryLocalStorageFirst) {
```

Comment:
Do we want to be explicit about what `options.__tryLocalStorageFirst` can be? This is truth-y, should it be `if (!options.__tryLocalStorageFirst === true)` instead? Edge case and a new option so no existing behavior but `"no"` or `"false"` would trigger a positive case here. Just wondering how this check is done elsewhere.

Reply:
done